app/main.py

```diff
@@ -2,7 +2,7 @@
 
 
 def move_file(command: str):
-    operator, current_file_name, destination = command.split()
+    operator, old_file_name, destination = command.split()
     *folders, new_file_name = destination.split("/")
 
     if operator != "mv":
```

Comment:
In my opinion, it's better to raise an error here.

Everything else is good !

Reply:
Yes, raising an error is better here but it is unnecessary in this task.